packages/labs/task/src/test/task_test.ts

```diff
@@ -225,6 +226,36 @@ suite('Task', () => {
     assert.equal(el.taskValue, `a1,b1`);
   });
 
+  test('tasks with args run when args change: deepEqual', async () => {
+    const el = getTestElement({
+      args: () => [[el.a], [el.b]],
```

Comment:
Recommend adding a comment to note the inner arrays, I almost missed that

Reply:
will do